app/helpers/authentication_helper.rb

```diff
@@ -0,0 +1,19 @@
+module AuthenticationHelper
+
+  def is_logged_in
+    return user_signed_in?
+  end
+
+  def can_access_employee_information_page
+    return user_signed_in?
+  end
+
+  def can_edit_weeks
+    return user_signed_in?
+  end
+
+  def can_add_weeks
+    return user_signed_in?
+  end
```

Comment:
`can_index_employees?`
`can_edit_weeks?`
`can_create_weeks?`

Also, you can remove the `return` since Ruby is awesome.